File: z101ytuonglangman/src/main/java/vn/loitp/app/activity/splash/SplashActivity.java
package vn.loitp.app.activity.splash;

import android.app.Activity;
import android.content.Intent;
import android.graphics.Color;
import android.os.Bundle;
import android.widget.ImageView;
import android.widget.TextView;

import com.google.android.gms.ads.InterstitialAd;

import loitp.basemaster.R;
import vn.loitp.app.activity.home.HomeMenuActivity;
import vn.loitp.core.base.BaseActivity;
import vn.loitp.core.utilities.LActivityUtil;
import vn.loitp.core.utilities.LUIUtil;

public class SplashActivity extends BaseActivity {
    private InterstitialAd interstitialAd;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        interstitialAd = LUIUtil.createAdFull(activity);
        LActivityUtil.hideSystemUI(getWindow().getDecorView());

        ImageView ivBkg = (ImageView) findViewById(R.id.iv_bkg);
        LUIUtil.setImageFromAsset(activity, "bkg.JPG", ivBkg);

        TextView tvAppName = (TextView) findViewById(R.id.tv_app_name);
        LUIUtil.setTextShadow(tvAppName, Color.WHITE);

        LUIUtil.setDelay(2500, new LUIUtil.DelayCallback() {
            @Override
            public void doAfter(int mls) {
                goToHome();
            }
        });
    }

    private void goToHome() {
        Intent intent = new Intent(activity, HomeMenuActivity.class);
        startActivity(intent);
        LActivityUtil.tranIn(activity);
        LUIUtil.displayInterstitial(interstitialAd, 100);
        finish();
    }

    @Override
    protected boolean setFullScreen() {
        return true;
    }

    @Override
    protected String setTag() {
        return getClass().getSimpleName();
    }

    @Override
    protected Activity setActivity() {
        return this;
    }

    @Override
    protected int setLayoutResourceId() {
        return R.layout.activity_splash;
    }
}

File: core/src/main/java/com/game/findnumber/SplashActivity.kt
package com.game.findnumber

import android.os.Bundle
import com.R
import com.annotation.IsFullScreen
import com.annotation.LogTag
import com.core.base.BaseFontActivity
import com.core.utilities.LUIUtil
import com.core.utilities.LValidateUtil

@LogTag("SplashActivity")
@IsFullScreen(false)
class SplashActivity : BaseFontActivity() {

    override fun setLayoutResourceId(): Int {
        return R.layout.l_activity_find_number_splash
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        setupViews()
        goToHome()
    }

    private fun setupViews() {
        LValidateUtil.isValidPackageName()
    }

    private fun goToHome() {
        LUIUtil.setDelay(mls = 1500, runnable = {
//            val intent = Intent(this, ComicLoginActivity::class.java)
//            startActivity(intent)
//            LActivityUtil.tranIn(this)
//            finish()
        })
    }

}
